packages/react/src/TextInput/TextInput.test.tsx

```diff
@@ -141,25 +136,25 @@ describe('TextInput', () => {
             </TextInput.Action>
           }
         />,
-      ),
+      ).container.firstChild,
     ).toMatchSnapshot()
   })
 
   it('renders trailingAction icon button', () => {
-    const handleAction = jest.fn()
+    const handleAction = vi.fn()
     expect(
       render(
         <TextInput
           name="search"
           placeholder={'Search'}
           trailingAction={<TextInput.Action onClick={handleAction} icon={SearchIcon} aria-label="Icon label" />}
         />,
-      ),
+      ).container.firstChild,
```

Comment:
remove all instances of `.container` and `.container.firstChild` in these tests for snapshot matching